wagtail/contrib/modeladmin/options.py

```diff
@@ -142,6 +142,7 @@ class ModelAdmin(WagtailRegisterable):
     inspect_view_extra_js = []
     form_view_extra_css = []
     form_view_extra_js = []
+    exclude_fields = []
```

Comment:
On the ModelAdmin class, we already have similar fields for InspectView (`inspect_view_fields` and `inspect_view_fields_exclude`). I'm wondering if `form_fields_exclude` might be more consistent?